### ISSUE TITLE
Typescript errors

### ISSUE THREAD

**Issue by olivierpascal** [staylor/react-helmet-async#92]
1.

```js
export interface FilledContext {
  helmet: HelmetData;
}; // ";" << Statements are not allowed in ambient contexts.
```

2.

```js
export class HelmetProvider extends React.Component<ProviderProps> {
  static canUseDOM = canUseDOM;
  // ^^ Initializers are not allowed in ambient contexts.
  // Cannot find name 'canUseDOM'. Did you mean the static member 'HelmetProvider.canUseDOM'?
}
```

**Comment by romansh-wixcom** [comment]
Duplicate #90 
Fixed in #91

**Comment by danilobuerger** [comment]
@romansh-wixcom Not fixed in #91. The first error is still there.

**Comment by staylor** [comment]
this is fixed here: #95 

https://www.npmjs.com/package/react-helmet-async/v/1.0.6
